### ISSUE TITLE
More Interactive and Detailed Read Me File 

### ISSUE THREAD

**Issue by Muskan-bhatt249** [MadhavKrishanGoswami/InkCode-Fusion#9]
Hi I can create a more detailed read me file, with all instructions on how to contribute to the project and details about the project as well as the  technologies used in it. Please provide a chance to contribute to the project.

**Comment by MadhavKrishanGoswami** [comment]
Absolutely! Feel free to set up the project locally and raise a pull request with any changes or improvements you think would be valuable!

**Comment by datasci-lohitt** [comment]
Hi Sir, @MadhavKrishanGoswami I have gone through the repository and the Readme file by you, I'll be able to write more detailed way of approaches, tools and Technologies used in the project.  **Please Assign me so that I can contribute to the project**

**Comment by MadhavKrishanGoswami** [comment]
@swarna-lohitt Absolutely, I'd love to have you contribute! Let's wait for @Muskan-bhatt249 for 12 hours to confirm if they're working on it. If not, you can go ahead and pick it up!

**Comment by Muskan-bhatt249** [comment]
Hi @MadhavKrishanGoswami I am working on it and would be summiting in by today it self, Thank you for patience.

**Comment by Shariq2003** [comment]
@MadhavKrishanGoswami please assign this to me

**Comment by MadhavKrishanGoswami** [comment]
Hey! Just checking in—has anyone started working on this issue yet?

**Comment by Shariq2003** [comment]
@MadhavKrishanGoswami I will raise PR before 12am positively

**Comment by Shariq2003** [comment]
@MadhavKrishanGoswami please check PR
